src/stores/useHolochainStore.js

```diff
@@ -124,7 +124,7 @@ const makeUseHolochainStore = ({ installed_app_id, app_ws_url, hc_admin_port })
       })
     },
     async fetchAgentKycLevel(_, __) {
-      const kycLevel = await hposHolochainCall({path: 'kyc', headers: {}, params: {}, method: 'get'})
+      const kycLevel = await hposHolochainCall({path: 'host/kyc', headers: {}, params: {}, method: 'get'})
```

Comment:
```suggestion
      const kycLevel = await hposHolochainCall({path: 'host/kyc_level', headers: {}, params: {}, method: 'get'})
```
Is this endpoint meant to point to the new kyc endpoint from hpos-api-rust?

If so, shouldn't this be `host/kyc_level`,  like referenced in the api integration tests: https://github.com/Holo-Host/hpos-api-rust/blob/develop/tests/integration.rs#L218

Reply:
Sheesh, yes, thank you. Fixed.